### PR TITLE
MDP-200: Adjust footer design

### PR DESCRIPTION
**Pull request by drerchi**

### CODE REVIEW

File: src/components/footer/Contacts.tsx [hunk 1]
```diff
@@ -4,15 +4,16 @@ import Email from "../../static/svg/email.svg";
...
+  <div className="ml-6 mt-2 lg:mt-0 sm:mt-0 mr-4">
     <div className="pb-1 flex justify-start items-center">
       <div className="mr-3 w-4">
         <Icon src={Phone} />
       </div>
       <div className="font-lato text-xs text-black">+421 905 271 220</div>
     </div>
     <div className="pb-1 flex justify-start items-center">
-      <div className="mr-3 w-8 lg:w-4">
+
```
Comment (pggl) [inline src/components/footer/Contacts.tsx]: remove space

**Review by pggl — CHANGES_REQUESTED** [review]

**Review by pggl — DISMISSED** [review]

File: src/style/custom.css [hunk 2]
```diff
@@ -20,8 +20,15 @@
...
-       top: 30px;
+       top: 20px;
     } 
+    .sm\:bg-oval{
+       background-image: url('../static/svg/ovals.svg');
+       width: 251px;
+       height: 301px;
+       top: 20px;
+    }
+
```
Comment (garySiladi) [inline src/style/custom.css]: the other classname selectors don't have empty lines between them. if that's the syntax convention, then remove this empty line

File: src/style/custom.css [hunk 3]
```diff
@@ -1,51 +1,60 @@
 @responsive{
-  .bg-cover-image{
-         background-image: url('../static/svg/yellowfooterbig.svg');
-         background-size: 100% 100%;
-         height: 473px;
+     .bg-cover-image {
```
Comment (garySiladi) [inline src/style/custom.css]: replace with tab

File: src/style/custom.css [hunk 4]
```diff
@@ -1,51 +1,60 @@
...
+      padding-bottom: 315px;
+    }
 
-   }
+    }
 
-.link {
-  @apply font-gilbert no-underline inline-block font-bold text-base text-black;
-}
+    .link {
```
Comment (garySiladi) [inline src/style/custom.css]: replace with tab

File: src/style/custom.css [hunk 5]
```diff
@@ -1,51 +1,60 @@
...
+      @apply font-gilbert no-underline inline-block font-bold text-base text-black;
+    }
 
-.userProfileLinks {
-  @apply font-lato no-underline text-black text-sm;
-}
+    .userProfileLinks {
```
Comment (garySiladi) [inline src/style/custom.css]: replace with tab

**Review by garySiladi — APPROVED** [review]

**Pull request merged**